packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -537,6 +537,7 @@ class ResidentWebRunner extends ResidentRunner {
             kNoneWorker,
             true,
             debuggingOptions.nativeNullAssertions,
+            null
```

Comment:
nit: trailing comma:
```suggestion
            null,
```